src/main/webapp/javascript/user-page.js

```diff
@@ -85,18 +87,39 @@ async function loadTypeRecipes(type) {
         addToListButton.style.display = "none";
         removeFromListButton.style.display = "inline-block";
       }
+      // sets up the icon of the other add to list button to a checkmark if it is contained
+      changeIcons(recipesDiv, other);
     }
   }
 }
 
+function changeIcons(recipesDiv, type) {
+    console.log("changeIcons");
```

Comment:
remove this? Or if you want to keep it for debugging purposes, looks like the indentation is too far right

Reply:
you're right that was for debugging